pangeo_forge_orchestrator/models.py

```diff
@@ -198,3 +198,27 @@ class RecipeRunReadWithBakeryAndFeedstock(RecipeRunRead):
 )
 
 MODELS = {"recipe_run": recipe_run_models, "bakery": bakery_models, "feedstock": feedstock_models}
+
+
+class APIKeyBase(SQLModel):
+    is_active: bool = Field(default=True)
+    is_admin: bool = Field(default=False)
+    created_at: datetime = Field(default_factory=datetime.now)
+
+
+class APIKeyCreate(SQLModel):
+    is_admin: bool = Field(default=False)
+
+
+class APIKeyRead(APIKeyBase):
+    pass
```

Comment:
Just noting that it appears the primary key (i.e. `encrypted_key`, in this model) is not included with the read response. This is divergent from the other models, which include their primary keys in the read response, but I assume is deliberate given the unique role of the `APIKey` table.

Reply:
Correct. There is never a need to share the encrypted key with the client. We may revise this as we build out the security model more.